src/components/main.tsx

```diff
@@ -68,6 +85,7 @@ const Main = () => {
                             <img src={MetaMask} alt="Connect with Metamask" />
                         </Button>
                     </InputGroupAddon>
+                    {!validAddress ? <FormFeedback>You will not be able to see this</FormFeedback> : null}
```

Comment:
What is this for? Not needed as far as I can tell

Reply:
to show them this address is not valid address? good UX? otherwise they wont understand why it is not working!

Reply:
Yeah, but this is not how FormFeedback works no? This won't show anything, you need to do form validation